src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -275,12 +274,15 @@ def build_cmd(self, env, host, **kwargs):
 
         index = kwargs.get('index', None)
 
+        daos_test_shared_dir = os.getenv('DAOS_TEST_SHARED_DIR',
+                                         os.getenv('HOME'))
+
         tst_vgd = " valgrind --xml=yes " + \
-                  "--xml-file={}/".format(self.log_path) + \
+                  "--xml-file={}/".format(daos_test_shared_dir) + \
                   r"valgrind.%q\{PMIX_ID\}.memcheck " + \
-                  "--fair-sched=try  --partial-loads-ok=yes " + \
+                  "--fair-sched=try --partial-loads-ok=yes " + \
                   "--leak-check=yes --gen-suppressions=all " + \
-                  "--suppressions=../etc/memcheck-cart.supp " + \
+                  "--suppressions=/etc/daos/memcheck-cart.supp " + \
```

Comment:
This will not work when run in a non-RPM based installation, such as when one installs to ./install/.

The local-dir test method should remain in working order so that developers don't **have** to install from RPMs.

Reply:
We did something similar for [DAOS-5566](https://github.com/daos-stack/daos/pull/4665) (_remove relative paths from test YAML configs_), so I've done the same for the valgrind .supp file.  9c34814